includes/features.php

```diff
@@ -129,6 +130,8 @@ function today_display_feature_horizontal( $post, $args=array() ) {
 	$is_linkable    = today_is_feature_linkable( $post );
 	$title          = wptexturize( $post->post_title );
 	$excerpt_length = ( $type === 'secondary' ) ? TODAY_SHORT_EXCERPT_LENGTH : TODAY_DEFAULT_EXCERPT_LENGTH;
+	// Explicitly overrwrite excerpt length if it's provided as an arg.
+	$excerpt_length = isset( $excerpt_len ) ? $excerpt_len : $excerpt_length;
```

Comment:
These variable names are a little confusing--something like `$custom_excerpt_length` instead of `$excerpt_len` might help with readability when determining what the final excerpt length is supposed to be here, and in `today_display_feature_vertical()`